Taskfile.yml

```diff
@@ -54,25 +54,29 @@ vars:
 
 tasks:
   default:
+    deps:
+      - quick-checks
+
+  quick-checks:
     desc: Perform all fast local pre-commit tasks.
     deps:
       - generator:quick-checks
       - controller:quick-checks
       - asoctl:quick-checks
-      - crossplane:generate-crds
```

Comment:
Isn't removing `crossplane:generate-crds` from `default` a behavior change? Is it intended?